ui/components/app/modals/convert-token-to-nft-modal/convert-token-to-nft-modal.js

```diff
@@ -47,16 +46,11 @@ const ConvertTokenToNFTModal = ({ hideModal, tokenAddress }) => {
       cancelText={t('cancel')}
     >
       <div className="convert-token-to-nft-modal">
-        <Typography
-          variant={TypographyVariant.H6}
```

Comment:
H6 is 14px but by the looks this text should be default size at 16px so no variant prop is required here